--- inventor/lib/database/include/machine.h.orig	2003-07-12 00:40:53.000000000 +0200
+++ inventor/lib/database/include/machine.h	2014-11-02 11:57:09.977452599 +0100
@@ -143,13 +143,56 @@
  * Linux i386/ia64 machine dependent setup
  */
 
-#if __i386__ || __ia64__
+#if __i386__ || __ia64__ || __x86_64__
 
 #define MACHINE_WORD_FORMAT	DGL_LITTLE_ENDIAN
 #define MACHINE_FLOAT_FORMAT	DGL_NON_IEEE
 
 #endif /* __i386__ || __ia64__ */
 
+/*
+ * Linux alpha machine dependent setup
+ */
+
+#ifdef __alpha__
+
+#define MACHINE_WORD_FORMAT DGL_LITTLE_ENDIAN
+#define MACHINE_FLOAT_FORMAT DGL_NON_IEEE
+
+#endif
+
+/*
+ * Linux ppc machine dependent setup
+ */
+
+#ifdef __powerpc__
+
+#define MACHINE_WORD_FORMAT DGL_BIG_ENDIAN
+#define MACHINE_FLOAT_FORMAT DGL_BIG_IEEE
+
+#endif
+
+/*
+ * Linux sparc machine dependent setup
+ */
+
+#ifdef __sparc__
+
+#define MACHINE_WORD_FORMAT DGL_BIG_ENDIAN
+#define MACHINE_FLOAT_FORMAT DGL_BIG_IEEE
+
+#endif
+
+/*
+ * Linux AMD64 machine dependent setup
+ */
+
+#ifdef __x86_64__
+
+#define MACHINE_WORD_FORMAT DGL_LITTLE_ENDIAN
+#define MACHINE_FLOAT_FORMAT DGL_NON_IEEE
+
+#endif
 
 /*
  * Apple Darwin (Mac OS X) machine dependent setup
@@ -321,7 +364,7 @@
  */
 
 #if MACHINE_FLOAT_FORMAT == DGL_NON_IEEE
-#if __i386__ || __ia64__
+#if __i386__ || __ia64__ || __x86_64__
 void mem_hton_float(float *t, float *f);
 void mem_ntoh_float(float *t, float *f);
 void mem_hton_double(double *t, double *f);
--- inventor/lib/database/src/so/SoType.c++.orig	Wed Mar 17 12:41:24 2004
+++ inventor/lib/database/src/so/SoType.c++	Wed Mar 17 13:10:58 2004
@@ -224,9 +224,17 @@
 #endif
 #else // __sgi
 #ifdef DEBUG
+#if defined(__x86_64__) || defined(__arch64__) || defined(__s390x__) || defined(__powerpc64__)
+	char *longestName = "/usr/lib64/InventorDSO/.so";
+#else
 	char *longestName = "/usr/lib/InventorDSO/.so";
+#endif
 #endif // DEBUG
+#if defined(__x86_64__) || defined(__arch64__) || defined(__s390x__) || defined(__powerpc64__)
+	const char *libDir = "lib64";
+#else
 	const char *libDir = "lib";
+#endif
 	const char *abiName = "";
 	//
 	// XXX Alex -- add additional layer of abstraction on top
