--- inventor/lib/database/src/sb/SbTime.c++.orig	Sat Oct 14 12:46:07 2000
+++ inventor/lib/database/src/sb/SbTime.c++	Sat Oct 12 21:47:15 2002
@@ -67,7 +67,9 @@
 // Use: public
 
 #ifndef __sgi
-inline static int trunc(double x) { return int(x); }
+inline static int trunci(double x) { return int(x); }
+#else
+#define trunci trunc
 #endif // !__sgi
 
 SbTime::SbTime(double sec)
@@ -75,7 +77,7 @@
 ////////////////////////////////////////////////////////////////////////
 {
     if (sec >= 0) {
-	t.tv_sec = trunc(sec);
+	t.tv_sec = trunci(sec);
 	t.tv_usec = (time_t) (0.5 + (sec - t.tv_sec) * 1000000.0);
     }
     else
--- inventor/apps/samples/widgets/componentTest.c++.orig	Sun Oct 29 16:04:17 2000
+++ inventor/apps/samples/widgets/componentTest.c++	Sat Oct 12 22:59:33 2002
@@ -59,7 +59,7 @@
     exit(0);
 }
 
-void
+int
 main(unsigned int argc, char *argv[])
 {
     if (argc != 2)
@@ -85,4 +85,5 @@
     XtRealizeWidget(mainWindow);
     
     SoXt::mainLoop();
+    return 0;
 }
--- inventor/apps/tools/ivview/ivview.c++.orig	Sat Oct 12 22:39:00 2002
+++ inventor/apps/tools/ivview/ivview.c++	Sat Oct 12 23:01:44 2002
@@ -1106,7 +1106,7 @@
 
 ///////////////////////////////////////////////////////////////////
 //
-void
+int
 main(int argc, char **argv)
 {
     XtAppContext appContext;
@@ -1169,4 +1169,5 @@
     setBusyCursor(FALSE);
     
     SoXt::mainLoop();
+    return 0;
 }
--- inventor/apps/demos/SceneViewer/SceneViewer.c++.orig	Sat Oct 14 12:46:05 2000
+++ inventor/apps/demos/SceneViewer/SceneViewer.c++	Sat Oct 12 23:03:36 2002
@@ -138,7 +138,7 @@
 }
 
 
-void main(int argc, char **argv)
+int main(int argc, char **argv)
 {
     Widget	 	mainWindow;
     SoSceneViewer	*sv;
@@ -230,4 +230,5 @@
     // Loop forever
     //
     SoXt::mainLoop();
+    return 0;
 }
--- inventor/apps/demos/SceneViewer/SoSceneViewer.c++.orig	Sat Oct 12 22:39:00 2002
+++ inventor/apps/demos/SceneViewer/SoSceneViewer.c++	Sat Oct 12 23:21:27 2002
@@ -364,6 +364,9 @@
     XtGetSubresources(  getParentWidget(), &lightData[0], "intl", "Intl",
                         intl_light_resources, intl_num_light_resources,
 			NULL, 0 );
+    XtGetSubresources(  getParentWidget(), &Light, "intl", "Intl",
+                        intl_light_resources2, intl_num_light_resources2,
+			NULL, 0 );
     XtGetSubresources(  getParentWidget(), &pulldownData[0], "intl", "Intl",
                         intl_pulldown_resources, intl_num_pulldown_resources,
 			NULL, 0 );
--- inventor/apps/demos/SceneViewer/SoSceneMenu.h.orig	Sun Oct 29 16:04:15 2000
+++ inventor/apps/demos/SceneViewer/SoSceneMenu.h	Sat Oct 12 23:20:55 2002
@@ -468,23 +468,27 @@
   { "ambientLighting", NULL, XtRString, sizeof(XtRString),
     (Cardinal) SV_OFFSET(&lightData[4].name, &lightData[0]),
     XtRString, (XtPointer) "Ambient Lighting...",},
+};
+int intl_num_light_resources = XtNumber( intl_light_resources );
+
+XtResource intl_light_resources2[] = {
   { "headlight", NULL, XtRString, sizeof(XtRString),
-    (Cardinal) SV_OFFSET(&Light.headlight, &lightData[0]),
+    (Cardinal) SV_OFFSET(&Light.headlight, &Light),
     XtRString, (XtPointer) "Headlight ",},
   { "editlight", NULL, XtRString, sizeof(XtRString),
-    (Cardinal) SV_OFFSET(&Light.editlight, &lightData[0]),
+    (Cardinal) SV_OFFSET(&Light.editlight, &Light),
     XtRString, (XtPointer) "Edit",},
   { "directionallight", NULL, XtRString, sizeof(XtRString),
-    (Cardinal) SV_OFFSET(&Light.directionallight, &lightData[0]),
+    (Cardinal) SV_OFFSET(&Light.directionallight, &Light),
     XtRString, (XtPointer) "Directional ",},
   { "spotlight", NULL, XtRString, sizeof(XtRString),
-    (Cardinal) SV_OFFSET(&Light.spotlight, &lightData[0]),
+    (Cardinal) SV_OFFSET(&Light.spotlight, &Light),
     XtRString, (XtPointer) "Spot ",},
   { "pointlight", NULL, XtRString, sizeof(XtRString),
-    (Cardinal) SV_OFFSET(&Light.pointlight, &lightData[0]),
+    (Cardinal) SV_OFFSET(&Light.pointlight, &Light),
     XtRString, (XtPointer) "Point ",},
 };
-int intl_num_light_resources = XtNumber( intl_light_resources );
+int intl_num_light_resources2 = XtNumber( intl_light_resources2 );
 
 static SoSceneViewerMenu pulldownData[] = {
 //  {name, 	id,	    	subMenu,    subItemCount}
--- inventor/apps/demos/drop/drop.c++.orig	Sat Oct 14 12:46:05 2000
+++ inventor/apps/demos/drop/drop.c++	Sat Oct 12 23:25:02 2002
@@ -69,7 +69,7 @@
  |
  _______________________________________________________________________
  */
-void
+int
 main( int, char **argv )
 {
     // Initialize Inventor and Xt
@@ -93,6 +93,7 @@
     // Enter the Xt event loop
     SoXt::show( appWindow );
     SoXt::mainLoop();
+    return 0;
 }
 
 
--- inventor/apps/demos/gview/gview.c++.orig	Sun Oct 29 16:04:15 2000
+++ inventor/apps/demos/gview/gview.c++	Sat Oct 12 23:26:15 2002
@@ -392,7 +392,7 @@
 //    Mainline.
 //
 
-void
+int
 main(int argc, char **argv)
 //
 ////////////////////////////////////////////////////////////////////////
--- inventor/apps/demos/maze/maze.c++.orig	Sun Oct 29 16:04:15 2000
+++ inventor/apps/demos/maze/maze.c++	Sat Oct 12 23:27:15 2002
@@ -1143,7 +1143,7 @@
     ra->setOverlaySceneGraph(logo);
 }
 
-void
+int
 main(int argc, char *argv[])
 {
     Widget              mainWindow;
@@ -1332,5 +1332,6 @@
 
     SoXt::show(mainWindow);
     SoXt::mainLoop();
+    return 0;
 }
 
--- inventor/apps/demos/noodle/noodle.c++.orig	Sun Oct 29 16:04:15 2000
+++ inventor/apps/demos/noodle/noodle.c++	Sat Oct 12 23:28:50 2002
@@ -280,7 +280,7 @@
     interface->fileQuitEvent();
 }
 
-void
+int
 main(int argc, char **argv)
 {
     // Parse command line arguments. This may fill in the 
@@ -680,5 +680,6 @@
     SoXt::show(mainWindow);
 
     SoXt::mainLoop();
+    return 0;
 }
 
--- inventor/apps/demos/textomatic/textomatic.c++.orig	Sat Oct 12 22:39:00 2002
+++ inventor/apps/demos/textomatic/textomatic.c++	Sat Oct 12 23:31:42 2002
@@ -403,7 +403,7 @@
 const int Vert2 = 60;
 Labels labels;        // Global variable.
 
-void
+int
 main(int argc, char **argv)
 {
     XtSetLanguageProc( NULL, _myXtDefaultLanguageProc, NULL );
@@ -541,4 +541,5 @@
 
     SoXt::show(w);
     SoXt::mainLoop();
+    return 0;
 }
