diff -Nur inventor.orig/apps/demos/revo/revo.c++ inventor/apps/demos/revo/revo.c++
--- inventor.orig/apps/demos/revo/revo.c++	Tue May 29 14:47:23 2001
+++ inventor/apps/demos/revo/revo.c++	Tue May 29 14:55:45 2001
@@ -87,7 +87,7 @@
     }
 
     char command[100];
-    sprintf(command, "which acroread >& /dev/null");
+    sprintf(command, "which acroread >/dev/null 2>&1");
 
     int err = system(command);
     if (err)
