--- inventor/lib/database/include/Inventor/SoPath.h.orig	2001-09-25 02:45:33.000000000 +0200
+++ inventor/lib/database/include/Inventor/SoPath.h	2014-11-02 13:00:07.363960823 +0100
@@ -79,6 +79,7 @@
 
 class SoPathList;
 class SoWriteAction;
+class SoTempPath;
 
 class SoPath : public SoBase {
 
--- inventor/lib/database/src/so/SoInput.c++.orig	2000-10-26 01:47:47.000000000 +0200
+++ inventor/lib/database/src/so/SoInput.c++	2014-11-02 13:02:31.810621428 +0100
@@ -2810,7 +2810,7 @@
 
     if (addToGlobalDict) {
 	// Look for the first '+':
-	char *firstPlus = strchr(n, '+');
+	const char *firstPlus = strchr(n, '+');
 
 	if (firstPlus == NULL) {
 	    base->setName(name);
--- inventor/lib/interaction/src/nodekits/SoInteractionKit.c++.orig	2000-08-15 14:56:25.000000000 +0200
+++ inventor/lib/interaction/src/nodekits/SoInteractionKit.c++	2014-11-02 13:28:26.740556171 +0100
@@ -854,8 +854,8 @@
 	    // (as in "childList[0].appearance")
 	    // If so, get the string up to whichever came first.
 	    // This will be the 'intermediary' we look for.
-	    char *dotPtr   = strchr( partName.getString(), '.' );
-	    char *brackPtr = strchr( partName.getString(), '[' );
+	    const char *dotPtr   = strchr( partName.getString(), '.' );
+	    const char *brackPtr = strchr( partName.getString(), '[' );
 
 	    if ( dotPtr != NULL || brackPtr != NULL ) {
 		char *nameCopy = strdup( partName.getString() );
