--- inventor/doc/help/GNUmakefile.orig	2014-11-02 09:16:19.687857662 +0100
+++ inventor/doc/help/GNUmakefile	2014-11-02 09:33:06.657815405 +0100
@@ -4,7 +4,7 @@
 all:
 
 install:
-	mkdir -p $(IVHELPDIR)/Inventor
-	for h in *.help; do $(INSTALL) -m 644 $$h $(IVHELPDIR)/Inventor; done
+	mkdir -p $(IVHELPDIR)
+	for h in *.help; do $(INSTALL) -m 644 $$h $(IVHELPDIR); done
 
 include $(IVCOMMONRULES)
--- inventor/make/ivcommondefs.orig	2014-11-02 09:16:19.687857662 +0100
+++ inventor/make/ivcommondefs	2014-11-02 09:37:19.434471464 +0100
@@ -192,13 +192,13 @@
 IVDLIBDIR = $(IVLIBDIR)/InventorDebug
 IVHDRTOP = $(IVROOT)/$(IVPREFIX)/include/Inventor
 IVHDRDIR = $(IVHDRTOP)/$(HDRSUBDIR)
-IVMAN1DIR = $(IVROOT)/$(IVPREFIX)/man/man1
-IVMAN3DIR = $(IVROOT)/$(IVPREFIX)/man/man3
-IVHELPDIR = $(IVROOT)/$(IVPREFIX)/share/help
-IVDATADIR = $(IVROOT)/$(IVPREFIX)/share/data/models
-IVDEMODIR = $(IVROOT)/$(IVPREFIX)/demos
-IVDEMOBINDIR = $(IVDEMODIR)/Inventor
-IVDEMODATADIR = $(IVDEMODIR)/data/Inventor
+IVMAN1DIR = $(IVROOT)/$(IVPREFIX)/share/man/man1
+IVMAN3DIR = $(IVROOT)/$(IVPREFIX)/share/man/man3
+IVHELPDIR = $(IVROOT)/$(IVPREFIX)/share/Inventor/help
+IVDATADIR = $(IVROOT)/$(IVPREFIX)/share/Inventor/data/models
+IVDEMODIR = $(IVROOT)/$(IVPREFIX)/share/Inventor/demos
+IVDEMOBINDIR = $(IVBINDIR)
+IVDEMODATADIR = $(IVDEMODIR)/data
 
 
 
--- inventor/make/ivcommonrules.orig	2003-07-12 00:40:54.000000000 +0200
+++ inventor/make/ivcommonrules	2014-11-02 15:50:32.616865176 +0100
@@ -119,7 +119,7 @@
 	  fi; \
 	fi
 	if [ -e $(DEMO).about ]; then \
-	  $(INSTALL) -m 644 $(DEMO).about $(IVDEMOBINDIR); \
+	  $(INSTALL) -m 644 $(DEMO).about $(IVHELPDIR); \
 	fi
 
 
--- inventor/lib/database/src/so/SoType.c++.orig	Sat Oct 14 12:46:07 2000
+++ inventor/lib/database/src/so/SoType.c++	Mon May 28 21:16:03 2001
@@ -283,11 +283,6 @@
 	    dsoHandle = sgidlopen_version(DSOFile, RTLD_LAZY, "sgi3.0", 0);
 	}	    
 	if (dsoHandle == NULL && !isRoot) {
-	    sprintf(DSOFile, "/usr/local/%s/InventorDSO/%s.so", 
-                                 libDir, nameChars);
-	    dsoHandle = sgidlopen_version(DSOFile, RTLD_LAZY, "sgi3.0", 0);
-	}	    
-	if (dsoHandle == NULL) {
 	    sprintf(DSOFile,"/usr/%s/InventorDSO/%s.so", libDir, nameChars);
 	    dsoHandle = sgidlopen_version(DSOFile, RTLD_LAZY, "sgi3.0", 0);
 	}	    
--- inventor/libSoXt/include/Inventor/Xt/SoXtComponent.h.orig	2014-11-02 09:16:19.694524329 +0100
+++ inventor/libSoXt/include/Inventor/Xt/SoXtComponent.h	2014-11-02 09:38:44.417801230 +0100
@@ -268,7 +268,7 @@
     // name. By default the file will be searched using:
     //	    1) current working directory
     //	    2) SO_HELP_DIR environment variable
-    //	    3) $(IVPREFIX)/share/help/Inventor
+    //	    3) $(IVPREFIX)/share/Inventor/help
     //	    4) else bring a no help card found dialog
     //
     void    openHelpCard(const char *cardName);
--- inventor/libSoXt/src/SoXtComponent.c++.orig	2014-11-02 09:16:19.694524329 +0100
+++ inventor/libSoXt/src/SoXtComponent.c++	2014-11-02 09:39:34.901132444 +0100
@@ -561,7 +561,7 @@
 //  card given the file name. By default the file will be searched using:
 //	    1) current working directory
 //	    2) SO_HELP_DIR environment variable
-//	    3) $(IVPREFIX)/share/help/Inventor
+//	    3) $(IVPREFIX)/share/Inventor/help
 //	    4) else bring a "No help card found message"
 //
 // Use: protected
@@ -622,7 +622,7 @@
     }
     
     // else check for the default location
-    strcpy(cardPath, IVPREFIX "/share/help/Inventor/");
+    strcpy(cardPath, IVPREFIX "/share/Inventor/help/");
     strcat(cardPath, cardName);
     if ( access(cardPath, R_OK) == 0 ) {
 	strcat(pgrCmd, cardPath);
--- inventor/libSoXt/src/mtlEdit/SoXtMtlEdit.c++.orig	2014-11-02 09:16:19.701190998 +0100
+++ inventor/libSoXt/src/mtlEdit/SoXtMtlEdit.c++	2014-11-02 09:40:19.134463921 +0100
@@ -156,7 +156,7 @@
 	"Material Emissive Color"
 };
 
-static char *mtlDir  = IVPREFIX "/share/data/materials";
+static char *mtlDir  = IVPREFIX "/share/Inventor/data/materials";
 
 static char *slider_labels[] = { (char *)NULL, (char *)NULL, (char *)NULL,
 				 (char *)NULL, (char *)NULL, (char *)NULL
--- inventor/libSoXt/src/mtlEdit/SoXtMtlList.c++.orig	2014-11-02 09:16:20.341190969 +0100
+++ inventor/libSoXt/src/mtlEdit/SoXtMtlList.c++	2014-11-02 09:40:35.281129910 +0100
@@ -264,7 +264,7 @@
     setClassName("SoXtMaterialList");
     if (dir != NULL)
 	materialDir = strdup(dir);
-    else materialDir = strdup(IVPREFIX "/share/data/materials");
+    else materialDir = strdup(IVPREFIX "/share/Inventor/data/materials");
 
     callbackList = new SoCallbackList;
     curPalette = -1;
--- inventor/libFL/ang/fl.c.orig	2014-11-02 09:16:20.341190969 +0100
+++ inventor/libFL/ang/fl.c	2014-11-02 09:42:05.567792788 +0100
@@ -95,7 +95,7 @@
   NULL
 };
 
-static char *fontPath = IVPREFIX "/share/data/fonts";
+static char *fontPath = IVPREFIX "/share/Inventor/data/fonts";
 int fl_debug = FALSE;
 
 /*
--- inventor/doc/man/man3/ivm/components/SoXtMaterialList.ivm.orig	Tue Aug 15 14:56:12 2000
+++ inventor/doc/man/man3/ivm/components/SoXtMaterialList.ivm	Mon May 28 23:22:05 2001
@@ -26,7 +26,7 @@
 directory for the material palettes. You can have any number of palettes
 in this directory. A palette is a subdirectory that contains Inventor data
 files, where each file describes one material. Predefined Inventor materials
-are found in /usr/share/data/materials.
+are found in /usr/share/Inventor/data/materials.
 }
 
 METHOD addCB addCallback(
--- inventor/apps/tools/ivview/ivview.c++.orig	2014-11-02 09:16:20.347857632 +0100
+++ inventor/apps/tools/ivview/ivview.c++	2014-11-02 09:42:49.531124277 +0100
@@ -715,9 +715,9 @@
 static void
 showAboutDialog()
 {
-   if (access(IVPREFIX "/share/help/ivview/ivview.about", R_OK) != 0) {
+   if (access(IVPREFIX "/share/Inventor/help/ivview/ivview.about", R_OK) != 0) {
         system("xmessage 'Sorry, could not find "
-               IVPREFIX "/share/help/ivview/ivview.about' > /dev/null");
+               IVPREFIX "/share/Inventor/help/ivview/ivview.about' > /dev/null");
         return;
     }
 
@@ -731,7 +731,7 @@
         return;
     }
 
-    sprintf(command, "acroread " IVPREFIX "/share/help/ivview/ivview.about &");
+    sprintf(command, "acroread " IVPREFIX "/share/Inventor/help/ivview/ivview.about &");
     system(command);
 }
 
--- inventor/apps/tools/ivview/GNUmakefile.orig	Mon May 28 18:45:38 2001
+++ inventor/apps/tools/ivview/GNUmakefile	Mon May 28 23:49:37 2001
@@ -12,7 +12,7 @@
 all: all_ivbin
 
 install: install_ivbin
-	mkdir -p $(IVHELPDIR)/$(PROGRAM)
-	$(INSTALL) -m 644 $(PROGRAM).about $(IVHELPDIR)/$(PROGRAM)
+	mkdir -p $(IVHELPDIR)
+	$(INSTALL) -m 644 $(PROGRAM).about $(IVHELPDIR)
 
 include $(IVCOMMONRULES)
--- inventor/apps/demos/SceneViewer/SoSceneViewer.c++.orig	2014-11-02 09:16:20.357857637 +0100
+++ inventor/apps/demos/SceneViewer/SoSceneViewer.c++	2014-11-02 09:54:20.107761965 +0100
@@ -4734,10 +4734,10 @@
 //
 ////////////////////////////////////////////////////////////////////////
 {
-    if (access(IVPREFIX "/demos/Inventor/SceneViewer.about", R_OK) != 0)
+    if (access(IVPREFIX "/share/Inventor/demos/data/SceneViewer.about", R_OK) != 0)
     {
 	system("xmessage 'Sorry, could not find "
-	       IVPREFIX "/demos/Inventor/SceneViewer.about' > /dev/null");
+	       IVPREFIX "/share/Inventor/demos/data/SceneViewer.about' > /dev/null");
 	return;
     }
 
@@ -4753,7 +4753,7 @@
     }
 
     sprintf(command, "acroread "
-            IVPREFIX "/demos/Inventor/SceneViewer.about &");
+            IVPREFIX "/share/Inventor/demos/data/SceneViewer.about &");
     system(command);
 }
 
--- inventor/apps/demos/gview/DisplayGraph.c++.orig	2014-11-02 10:18:30.964367742 +0100
+++ inventor/apps/demos/gview/DisplayGraph.c++	2014-11-02 10:24:00.081020597 +0100
@@ -67,7 +67,7 @@
 SbDict	*DisplayGraph::iconDict;
 
 #define ICON_FILE	"gviewIcons.iv"
-#define ICON_INST_DIR	IVPREFIX "/demos/data/Inventor"
+#define ICON_INST_DIR	IVPREFIX "/share/Inventor/demos/data"
 #define ICON_ENV_VAR	"IV_GRAPH_DIR"
 
 ////////////////////////////////////////////////////////////////////////
--- inventor/apps/demos/noodle/Interface.c++.orig	2014-11-02 09:16:20.364524304 +0100
+++ inventor/apps/demos/noodle/Interface.c++	2014-11-02 10:05:25.657734032 +0100
@@ -399,9 +399,9 @@
 void
 Interface::showAboutDialog()
 {
-   if (access(IVPREFIX "/demos/Inventor/noodle.about", R_OK) != 0) {
+   if (access(IVPREFIX "/share/Inventor/demos/data/noodle.about", R_OK) != 0) {
         system("xmessage 'Sorry, could not find "
-               IVPREFIX "/demos/Inventor/noodle.about' > /dev/null");
+               IVPREFIX "/share/Inventor/demos/data/noodle.about' > /dev/null");
         return;
     }
 
@@ -415,7 +415,7 @@
         return;
     }
 
-    sprintf(command, "acroread " IVPREFIX "/demos/Inventor/noodle.about &");
+    sprintf(command, "acroread " IVPREFIX "/share/Inventor/demos/data/noodle.about &");
     system(command);
 }
 
--- inventor/apps/demos/qmorf/qmorf.c++.orig	2014-11-02 09:16:20.371190968 +0100
+++ inventor/apps/demos/qmorf/qmorf.c++	2014-11-02 10:07:27.381062257 +0100
@@ -414,7 +414,7 @@
 		"they must contain QuadMeshes (qmorf can\n");
 	fprintf(stderr, "only morph QuadMesh nodes).\n");
 	fprintf(stderr, "The directory "
-		IVPREFIX "/share/data/models/CyberHeads "
+		IVPREFIX "/share/Inventor/data/models/CyberHeads "
 		"contains good data to morph.\n");
 
 	exit(7);
@@ -436,10 +436,10 @@
 void
 showAboutDialog(Widget, XtPointer, XtPointer)
 {
-    if (access(IVPREFIX "/demos/Inventor/qmorf.about", R_OK) != 0)
+    if (access(IVPREFIX "/share/Inventor/demos/data/qmorf.about", R_OK) != 0)
     {
 	system("xmessage 'Sorry, could not find "
-	       IVPREFIX "/demos/Inventor/qmorf.about' > /dev/null");
+	       IVPREFIX "/share/Inventor/demos/data/qmorf.about' > /dev/null");
 	return;
     }
     char command[100];
@@ -453,7 +453,7 @@
 	return;
     }
 
-    sprintf(command, "acroread " IVPREFIX "/demos/Inventor/qmorf.about &");
+    sprintf(command, "acroread " IVPREFIX "/share/Inventor/demos/data/qmorf.about &");
     system(command);
 }	
 
--- inventor/apps/demos/revo/revo.c++.orig	2014-11-02 09:16:20.374524299 +0100
+++ inventor/apps/demos/revo/revo.c++	2014-11-02 10:08:17.104393504 +0100
@@ -78,10 +78,10 @@
 void
 showAboutDialog(Widget, XtPointer, XtPointer)
 {
-    if (access(IVPREFIX "/demos/Inventor/revo.about", R_OK) != 0)
+    if (access(IVPREFIX "/share/Inventor/demos/data/revo.about", R_OK) != 0)
     {
 	system("xmessage 'Sorry, could not find "
-	       IVPREFIX "/demos/Inventor/revo.about' > /dev/null");
+	       IVPREFIX "/share/Inventor/demos/data/revo.about' > /dev/null");
 	return;
     }
 
@@ -96,7 +96,7 @@
 	return;
     }
 
-    sprintf(command, "acroread " IVPREFIX "/demos/Inventor/revo.about &");
+    sprintf(command, "acroread " IVPREFIX "/share/Inventor/demos/data/revo.about &");
     system(command);
 }	
 
--- inventor/apps/demos/textomatic/textomatic.c++.orig	2014-11-02 09:16:20.374524299 +0100
+++ inventor/apps/demos/textomatic/textomatic.c++	2014-11-02 10:08:52.151058700 +0100
@@ -160,10 +160,10 @@
 void
 showAboutDialog(Widget, XtPointer, XtPointer)
 {
-    if (access(IVPREFIX "/demos/Inventor/textomatic.about", R_OK) != 0)
+    if (access(IVPREFIX "/share/Inventor/demos/data/textomatic.about", R_OK) != 0)
     {
 	system("xmessage 'Sorry, could not find "
-	       IVPREFIX "/demos/Inventor/textomatic.about' > /dev/null");
+	       IVPREFIX "/share/Inventor/demos/data/textomatic.about' > /dev/null");
 	return;
     }
 
@@ -177,7 +177,7 @@
 	return;
     }
 
-    sprintf(command, "acroread " IVPREFIX "/demos/Inventor/textomatic.about &");
+    sprintf(command, "acroread " IVPREFIX "/share/Inventor/demos/data/textomatic.about &");
     system(command);
 }	
 
--- inventor/apps/samples/widgets/MyTextureEd.c++.orig	2014-11-02 09:16:20.381190965 +0100
+++ inventor/apps/samples/widgets/MyTextureEd.c++	2014-11-02 10:11:13.424386104 +0100
@@ -228,7 +228,7 @@
 
 
 static char *customTextureDir = ".textures";
-static char *defaultDir = IVPREFIX "/share/data/textures";
+static char *defaultDir = IVPREFIX "/share/Inventor/data/textures";
 static char *editorTitle = "Texture Editor";
 static char *noFileNameStr = "<empty>";
 
