--- inventor/libSoXt/GNUmakefile.orig	Sat Oct 14 12:46:08 2000
+++ inventor/libSoXt/GNUmakefile	Fri May 25 21:24:10 2001
@@ -5,7 +5,7 @@
 
 LLDLIBS = \
 	-lXm -lXt -lXext -lXi -lX11 -lm \
-	-lInventor -lGLU -lGL
+	-L../lib -lInventor -lGLU -lGL
 
 LLDLIBS += $(shell [ -e $(X11LIBDIR)/libXp.so ] && echo -lXp)
 
--- inventor/doc/man/ivman/GNUmakefile.orig	2014-11-02 09:04:07.224555065 +0100
+++ inventor/doc/man/ivman/GNUmakefile	2014-11-02 09:06:34.431215559 +0100
@@ -5,7 +5,7 @@
 
 CXXFILES = Main.c++ ClassDef.c++ Man.c++ Reader.c++ Writer.c++ BuildIssues.c++
 
-LLDLIBS = -lInventor
+LLDLIBS = -L../../../lib -lInventor
 
 all install: $(PROGRAM)
 
--- inventor/apps/samples/widgets/GNUmakefile.componentTest.orig	Sun Oct 29 16:04:16 2000
+++ inventor/apps/samples/widgets/GNUmakefile.componentTest	Sat May 26 03:55:48 2001
@@ -5,7 +5,7 @@
 
 CXXFILES = componentTest.c++
 
-LLDLIBS = libInventorWidget.a -lInventor -lInventorXt -lGLw
+LLDLIBS = libInventorWidget.a -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGLw
 
 all install: all_ivbin
 
--- inventor/apps/tools/ivcat/GNUmakefile.orig	Mon Sep 18 11:41:03 2000
+++ inventor/apps/tools/ivcat/GNUmakefile	Mon May 28 14:39:57 2001
@@ -5,7 +5,7 @@
 
 CXXFILES = ivcat.c++
 
-LLDLIBS = -lInventor
+LLDLIBS = -L../../../lib -lInventor
 
 all: all_ivbin
 
--- inventor/apps/tools/ivfix/GNUmakefile.orig	Mon May 28 14:40:38 2001
+++ inventor/apps/tools/ivfix/GNUmakefile	Mon May 28 14:40:14 2001
@@ -22,7 +22,7 @@
 	IfTypes.c++	\
 	IfWeeder.c++
 
-LLDLIBS = -lInventor
+LLDLIBS = -L../../../lib -lInventor
 
 all: all_ivbin
 
--- inventor/apps/tools/ivinfo/GNUmakefile.orig	Mon Sep 18 11:41:03 2000
+++ inventor/apps/tools/ivinfo/GNUmakefile	Mon May 28 14:40:54 2001
@@ -5,7 +5,7 @@
 
 CXXFILES = ivinfo.c++
 	
-LLDLIBS = -lInventor
+LLDLIBS = -L../../../lib -lInventor
 
 all: all_ivbin
 
--- inventor/apps/tools/ivview/GNUmakefile.orig	Tue Nov 14 03:49:52 2000
+++ inventor/apps/tools/ivview/GNUmakefile	2014-11-02 13:47:07.040509433 +0100
@@ -7,7 +7,7 @@
 
 LLDOPTS += -L../../samples/widgets
 
-LLDLIBS = -lInventorWidget -lInventorXt -lInventor
+LLDLIBS = -lInventorWidget -L../../../libSoXt -lInventorXt -L../../../lib -lInventor -lGL -lXm -lXt -lX11
 
 all: all_ivbin
 
--- inventor/apps/converters/ivdowngrade/GNUmakefile.orig	Mon Sep 18 11:40:59 2000
+++ inventor/apps/converters/ivdowngrade/GNUmakefile	Mon May 28 14:44:18 2001
@@ -22,7 +22,7 @@
 	SoText2V2.c++ \
 	SoText3V2.c++
 
-LLDLIBS = -lInventor
+LLDLIBS = -L../../../lib -lInventor
 
 all: all_ivbin
 
--- inventor/apps/nodes/BarChart/GNUmakefile.orig	Mon Sep 18 11:41:01 2000
+++ inventor/apps/nodes/BarChart/GNUmakefile	Mon May 28 14:45:17 2001
@@ -5,7 +5,7 @@
 
 CXXFILES = BarChart.c++
 
-LLDLIBS = -lInventor -lInventorXt
+LLDLIBS = -L../../../lib -lInventor -L../../../libSoXt -lInventorXt
 
 all install: all_ivbin
 
--- inventor/apps/nodes/Decal/GNUmakefile.orig	Mon Sep 18 11:41:01 2000
+++ inventor/apps/nodes/Decal/GNUmakefile	Mon May 28 14:45:40 2001
@@ -5,7 +5,7 @@
 
 CXXFILES = Decal.c++
 
-LLDLIBS = -lInventor -lInventorXt
+LLDLIBS = -L../../../lib -lInventor -L../../../libSoXt -lInventorXt
 
 all install: all_ivbin
 
--- inventor/apps/nodes/GeneralizedCylinder/GNUmakefile.orig	Mon Sep 18 11:41:01 2000
+++ inventor/apps/nodes/GeneralizedCylinder/GNUmakefile	Mon May 28 14:45:59 2001
@@ -7,7 +7,7 @@
 	GeneralizedCylinder.c++ Triangulator.c++ \
 	NurbMaker.c++
 
-LLDLIBS = -lInventor -lInventorXt
+LLDLIBS = -L../../../lib -lInventor -L../../../libSoXt -lInventorXt
 
 all install: all_ivbin
 
--- inventor/apps/demos/SceneViewer/GNUmakefile.orig	Mon Sep 18 11:40:59 2000
+++ inventor/apps/demos/SceneViewer/GNUmakefile	2014-11-02 13:56:24.263818989 +0100
@@ -8,7 +8,7 @@
 
 LLDOPTS += -L../../samples/widgets
 
-LLDLIBS = -lInventorWidget -lInventor -lInventorXt
+LLDLIBS = -lInventorWidget -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGL -lXm -lXt -lX11
 
 all: all_ivbin
 
--- inventor/apps/demos/drop/GNUmakefile.orig	Mon Sep 18 11:40:59 2000
+++ inventor/apps/demos/drop/GNUmakefile	2014-11-02 14:12:44.797111291 +0100
@@ -8,7 +8,7 @@
 	TsBlock.c++ TsPiece.c++ TsPieces.c++ TsField.c++ \
 	TsTrickle.c++ TsEvents.c++ TsRemoval.c++ TsViewer.c++ drop.c++
 
-LLDLIBS = -lInventor -lInventorXt
+LLDLIBS = -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGL -lXm -lXt -lX11
 
 all: all_ivbin
 
--- inventor/apps/demos/gview/GNUmakefile.orig	Sun Oct 29 16:04:15 2000
+++ inventor/apps/demos/gview/GNUmakefile	2014-11-02 14:19:19.057094717 +0100
@@ -8,7 +8,7 @@
 	gview.c++ DisplayGraph.c++ Error.c++ FieldEditor.c++ \
 	GraphIcon.c++ GraphViewer.c++ NodeCreator.c++
 
-LLDLIBS = -lInventor -lInventorXt
+LLDLIBS = -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGL -lXm -lXt -lX11
 
 all: all_ivbin
 
--- inventor/apps/demos/maze/GNUmakefile.orig	Mon Sep 18 11:41:00 2000
+++ inventor/apps/demos/maze/GNUmakefile	2014-11-02 14:27:50.883739935 +0100
@@ -8,7 +8,7 @@
 
 CXXFILES = maze.c++
 
-LLDLIBS = -lInventor -lInventorXt
+LLDLIBS = -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGL
 
 all: all_ivbin
 
--- inventor/apps/demos/noodle/GNUmakefile.orig	Tue Nov 14 03:49:52 2000
+++ inventor/apps/demos/noodle/GNUmakefile	2014-11-02 14:35:34.837053797 +0100
@@ -16,7 +16,7 @@
 	../../nodes/GeneralizedCylinder/NurbMaker.o \
 	../../nodes/GeneralizedCylinder/Triangulator.o \
 	../../nodes/GeneralizedCylinder/GeneralizedCylinder.o \
-	-lInventor -lInventorXt
+	-L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lXm -lXt -lX11
 
 all: all_ivbin
 
--- inventor/apps/demos/qmorf/GNUmakefile.orig	Mon Sep 18 11:41:00 2000
+++ inventor/apps/demos/qmorf/GNUmakefile	2014-11-02 14:45:41.077028356 +0100
@@ -6,7 +6,7 @@
 
 CXXFILES = QuadThing.c++ Background.c++ FlashBackground.c++
 
-LLDLIBS = -lInventor -lInventorXt
+LLDLIBS = -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGL -lXm -lXt
 
 all: all_ivbin
 
--- inventor/apps/demos/revo/GNUmakefile.orig	Mon Sep 18 11:41:00 2000
+++ inventor/apps/demos/revo/GNUmakefile	2014-11-02 14:47:18.593690930 +0100
@@ -10,7 +10,7 @@
 
 LLDOPTS += -L../../samples/common
 
-LLDLIBS = -lcommon -lInventor -lInventorXt
+LLDLIBS = -lcommon -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGL -lXm -lXt
 
 all: all_ivbin
 
--- inventor/apps/demos/textomatic/GNUmakefile.orig	Mon Sep 18 11:41:00 2000
+++ inventor/apps/demos/textomatic/GNUmakefile	2014-11-02 14:48:04.787022325 +0100
@@ -8,7 +8,7 @@
 
 LLDOPTS += -L../../samples/common
 
-LLDLIBS = -lcommon -lInventor -lInventorXt
+LLDLIBS = -lcommon -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGL -lXm -lXt -lX11
 
 all: all_ivbin
 
--- inventor/apps/samples/widgets/GNUmakefile.componentTest.orig	2014-11-02 13:29:26.153887014 +0100
+++ inventor/apps/samples/widgets/GNUmakefile.componentTest	2014-11-02 13:38:47.050530228 +0100
@@ -5,7 +5,7 @@
 
 CXXFILES = componentTest.c++
 
-LLDLIBS = libInventorWidget.a -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGLw
+LLDLIBS = libInventorWidget.a -L../../../lib -lInventor -L../../../libSoXt -lInventorXt -lGLw -lGL -lXm -lXt -lX11
 
 all install: all_ivbin
 
